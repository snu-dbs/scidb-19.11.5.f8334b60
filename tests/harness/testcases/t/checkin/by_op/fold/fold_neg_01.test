--setup
--start-query-logging

--test

# Chunk interval in last dimension must cover dimension interval
--error --code scidb::SCIDB_SE_OPERATOR::SCIDB_LE_CHUNK_INTERVAL_TOO_SMALL \
    "_fold(build(<v:int64>[i=0:9;j=0:1:0:1],i+j+1))"

--cleanup
--stop-query-logging
