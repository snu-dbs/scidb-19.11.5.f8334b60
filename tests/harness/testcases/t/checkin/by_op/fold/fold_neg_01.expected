SCIDB QUERY : <_fold(build(<v:int64>[i=0:9;j=0:1:0:1],i+j+1))>
[An error expected at this place for the query "_fold(build(<v:int64>[i=0:9;j=0:1:0:1],i+j+1))". And it failed with error code = scidb::SCIDB_SE_OPERATOR::SCIDB_LE_CHUNK_INTERVAL_TOO_SMALL. Expected error code = scidb::SCIDB_SE_OPERATOR::SCIDB_LE_CHUNK_INTERVAL_TOO_SMALL.]

